.github/ISSUE_TEMPLATE/4_KIT_release_checks.md

```diff
@@ -5,23 +5,31 @@ title: "[KIT NAME] Release Checks"
 labels: kit
 ---
 
-This issue tracks all relevant release checks for KITs.
+This issue tracks all relevant release checks for your KIT.
+
+> [!IMPORTANT]  
+> Follow the [guidance on how to use the templates](https://github.com/eclipse-tractusx/sig-release/blob/main/README.md#release-management-acceptance-criteria).
 
 ## Release Info
 
 Please provide information on what you want to be included in the Eclipse Tractus-X release.
 If you are not owner of this issue, please provide the information as comment to the issue.
+Make sure to assign this issue to expert(s) for their approval, as soon as you have finished preparation.
+Best practice: have a professional Peer review your prepared content before you submit for expert approvals.
```

Comment:
Suggestion: Have your prepared content reviewed by a professional peer group before submitting it for committer approval.

Reply:
text ok for me